README.md

```diff
@@ -45,6 +45,10 @@ If you have trouble getting pipenv to resolve python 3.8.1, you can also try:
 ```
 pipenv install --python ~/.pyenv/versions/3.6.10/bin/python3.6 --dev
 ```
+If you have trouble installing pyodbc, you can try:
```

Comment:
This comment us under the mac os heading but specific to Ubuntu - please put it under an appropriate heading, or clarify that it relates directly to ubuntu.

Reply:
Unit tests fixed.
Updated readme.
Pushed updates to script.